### ISSUE TITLE
训练参数，结果讨论

### ISSUE THREAD

**Issue by selfboot** [RVC-Boss/GPT-SoVITS#77]
可以贴下训练好的语音和原始数据集语音，看看大家的效果如何吗？

最好说下样本时长多少，微调的参数怎么设置，比如多少轮这样。

**Comment by CloudTronUSA** [comment]
@selfboot 参考我在这里发的：
https://github.com/RVC-Boss/GPT-SoVITS/issues/51#issuecomment-1901920999

**Comment by selfboot** [comment]
10 分钟收音器录制的高清音频，然后训练的时候默认参数，最后效果还可以，问了几个人，说有点像了。

当然录音的时候，容易有朗读的感觉，不像平时说话，这个貌似也有点难纠正过来。

**Comment by CloudTronUSA** [comment]
确实。主要是合成有一定的电子噪音和GPT喜欢漏字或者错字，朗读的问题也有但是我训练数据比较多所以还是不太明显的。在电子噪音这方面，还是BV2比较强（原神那个几乎没有），但语气语调还是这个厉害
